src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj

```diff
@@ -1,22 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.Batch</PackageId>
-		<Description>Provides management capabilities for Azure Batch service accounts.</Description>
-		<AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
-		<AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
-		<Version>6.0.0</Version>
-		<PackageTags>Microsoft Azure batch management;batch;</PackageTags>
-		<PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-management-changelog</PackageReleaseNotes>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-	</PropertyGroup>
+  <PropertyGroup>
```

Comment:
nit: please fix indentation

Reply:
This is fixed indentation - the indentation should be spaces, not tabs, right? It was tabs and I changed it to spaces.